menuboot.html

```diff
@@ -0,0 +1,129 @@
+<!doctype html>
+<html lang="en">
+  <head>
+  <title>Page Title</title>
+  <link rel="stylesheetboot" type="text/css" href="styleboot.css">
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+<link href="https://stackpath.bootstrapcdn.com/bootswatch/4.3.1/minty/bootstrap.min.css" rel="stylesheet" integrity="sha384-9NlqO4dP5KfioUGS568UFwM3lbWf3Uj3Qb7FBHuIuhLoDp3ZgAqPE1/MYLEBPZYM" crossorigin="anonymous">
+
+  </head>
+  <body>
+
+<a href="about.html" class="badge badge-primary">ABOUT US</a>
+<a href="menu.html" class="badge badge-secondary">MENU</a>
```

Comment:
In fact, only one page out of 5 is using bootstrap! There should all use and be uniform in style.
You also do not use navbars, tabs and html grids.